fe/fe-core/src/main/java/org/apache/doris/datasource/InternalDataSource.java

```diff
@@ -1731,6 +1726,26 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws UserExcep
         // this should be done before create partition.
         Map<String, String> properties = stmt.getProperties();
 
+        // get use light schema change
+        Boolean useLightSchemaChange = true;
```

Comment:
default should be false？

Reply:
Actually it wouldn't have any effect cause useLightSchemaChange would be assigned value by PropertyAnalyzer.analyzeUseLightSchemaChange which is set to false by default. But it's confusing indeed. Will change it soon.